api/scpca_portal/models/dataset.py

```diff
@@ -188,6 +215,22 @@ def original_files(self) -> Iterable[OriginalFile]:
     def original_file_paths(self) -> Set[Path]:
         return {Path(of.s3_key) for of in self.original_files}
 
+    def get_hash_data(self) -> int:
+        original_files_sorted = sorted(self.original_files, key=lambda of: of.s3_key)
```

Comment:
Since `self.original_files` is a queryset could we just get a sorted `values_list` instead of doing this after executing the query?